guake/guake_app.py

```diff
@@ -1004,6 +1005,20 @@ def is_using_unity(self):
                 return True
         return False
 
+    def is_using_gnome(self):
+        linux_distrib = platform.linux_distribution()
+        if linux_distrib[0].lower() != "ubuntu":
```

Comment:
Is this function's logic correct? I'm sure there's distros other than ubuntu out there that use GNOME, and I think ubuntu switched back to gnome shell at some point later. Is there no better way to check for gnome shell?